lib/SQLiteCpp

```diff
@@ -1 +1 @@
-Subproject commit 7e9983b00fb44c2e569ba54bfee6f548496d7f61
+Subproject commit af78d59349fd013341746b345961e3f19ecae60d
```

Comment:
Yes you are right. @SafwatHalaby if you do git status, does the file still appears? If yes add and commit it and rebase your commit and then the file should be removed from your commit.